cms/djangoapps/contentstore/tests/test_courseware_index.py

```diff
@@ -1275,8 +1275,8 @@ def _get_index_values_from_call_args(self, mock_index):
         Return content values from args tuple in a mocked calls list.
         """
         kall = mock_index.call_args
-        args, kwargs = kall  # pylint: disable=unused-variable
-        return args[0]
+        indexed_content, kwargs = kall  # pylint: disable=unused-variable
+        return indexed_content[0]
```

Comment:
Sergey, that's not what I meant. It was this `args, kwargs = kall` that did not raise my suspicions. I want to understand what is behind the zero index.

Reply:
list of json objects which generated by edx-search search results

Reply:
Can you make the unpacking smarter so that there are no questions about what is at index 0?